core/src/main/java/org/eclipse/hono/util/CredentialsConstants.java

```diff
@@ -8,34 +8,25 @@
  *
  * Contributors:
  *    Bosch Software Innovations GmbH - initial creation
- */package org.eclipse.hono.util;
+ */
+package org.eclipse.hono.util;
 
 import io.vertx.core.json.JsonObject;
-import io.vertx.proton.ProtonHelper;
-import org.apache.qpid.proton.amqp.Symbol;
-import org.apache.qpid.proton.amqp.messaging.AmqpValue;
-import org.apache.qpid.proton.amqp.messaging.ApplicationProperties;
-import org.apache.qpid.proton.amqp.messaging.MessageAnnotations;
 import org.apache.qpid.proton.message.Message;
 
 import java.util.*;
 
-import static org.eclipse.hono.util.MessageHelper.*;
-
 /**
  * Constants &amp; utility methods used throughout the Credentials API.
  */
-
-public final class CredentialsConstants {
+public final class CredentialsConstants extends RequestResponseApiConstants {
     /* registration actions */
     public static final String OPERATION_GET                     = "get";
     public static final String OPERATION_ADD                     = "add";
     public static final String OPERATION_UPDATE                  = "update";
     public static final String OPERATION_REMOVE                  = "remove";
 
     /* message payload fields */
-    public static final String FIELD_PAYLOAD                     = "payload";
-    public static final String FIELD_ENABLED                     = "enabled";
     public static final String FIELD_TYPE                        = "type";
     public static final String FIELD_AUTH_ID                     = "auth-id";
     public static final String FIELD_DEVICE_ID                   = "device-id";
```

Comment:
this is the same as `RegistrationConstants.FIELD_HONO_ID`. Please pull up and use consistently.